index.html

```diff
@@ -2322,6 +2258,169 @@ <h2>JSON-LD</h2>
 
       </section>
 
+        <section>
+            <h2>JSON Web Token</h2>
+
+            <p>
+                The verifiable credential data model as described in Section <a href="#core-data-model"></a> can be
+                encoded as a JSON Web Token (JWT) [[!RFC7519]] by doing the following:
+            </p>
+
+            <ul>
+                <li>The signature of the JWT is produced by the issuer of the verifiable credential.</li>
+                <li>Because JWS implicitly provides a verifiable proof of the contained verifiable credentials, the "proof" is no longer needed. Please note that
+                    the "jti" claim represents a nonce value.</li>
+                <li>The IANA JWT claims registry should be enforced whenever possible:</li>
+                <ul>
+                    <li>The "expirationDate" becomes the JWT "exp" claim (epoch time).</li>
+                    <li>The "id" of the verifiable credential becomes the JWT "jti" claim.</li>
+                    <li>The "id" contained in the "claim" object becomes the JWT "sub".</li>
+                    <li>If "created" and "creator" are required, these claims could be added to the JWT payload.</li>
+                    <li>The "issuer" becomes the JWT "iss" claim.</li>
+                    <li>The "issuanceDate" becomes the JWT "iat" claim.</li>
```

Comment:
Where is the nonce and domain specified? If you don't do that, the message can be replayed. This is confusing layers of abstraction... the VC Data Model spec isn't supposed to talk about the crypto... I don't think the group is chartered to do that. We may need to find non-normative weasel wording to make these sorts of assertions since certain W3C Member companies requested that we NOT make statements like the above.

Reply:
True, `nonce` has also to be included. Domain would be the `iss` claim. I am now aware of the group charter. It would be good to have a non-normative example similar to the JSON LD representation where the `nonce` is part of the `proof` attribute.

Reply:
I added the `nonce`to the VP and VC examples. Please note, the `jti`serves as a nonce in the VC already. In general, we don't talk about specific risk mitigation strategies and everybody is free to implement their own means using what the data model allows.